choreolib/src/main/java/choreo/trajectory/DifferentialSample.java

```diff
@@ -17,7 +17,7 @@ public class DifferentialSample implements TrajectorySample<DifferentialSample>
       Choreo.getProjectFile().config.differentialTrackWidth.val;
 
   /** The timestamp of this sample, relative to the beginning of the trajectory. */
-  public final double timestamp;
+  public final double t;
```

Comment:
Is there a way to work around having to name this t?

Reply:
Is it due to serialization or something?

Reply:
I think naming is due to serialisation

Reply:
All samples have a getTimestamp method aswell so i decided it was easier to change the name of the field in java opposed to everywhere else because there was already a clear way to get timestamp. I think we should reconsider trying to save space on the .traj files with shorter keys.